storage/s3.go

```diff
@@ -366,17 +379,8 @@ func (s *S3) Get(
 	concurrency int,
 	partSize int64,
 ) (int64, error) {
-	if concurrency == 1 {
```

Comment:
Why this block is removed?

Reply:
I did some refactoring so that passing all storage logic to `storage` is done while keeping interface size as small as possible